e2e/app/page/notebook-page.ts

```diff
@@ -58,11 +60,11 @@ export default class NotebookPage extends NotebookFrame {
     try {
       await this.findRunButton(120000);
     } catch (err) {
-      console.warn(`Reloading "${this.documentTitle}" because cannot find the Run button`);
+      logger.info(`Reloading "${this.documentTitle}" because cannot find the Run button`);
       await this.page.reload();
     }
-    // When open notebook for the first time, notebook websocket can close unexpectedly that causes kernel disconnect unexpectedly.
```

Comment:
is this text still true?  we should keep it if so.

Reply:
websocket closed is one cause. There are other reasons why notebook connection can fail. I think it's better if  don't mention websocket so that we don't mislead.

Reply:
ok